core/include/initcall.h

```diff
@@ -101,6 +101,31 @@ struct initcall {
 
 #define boot_final(fn)			__define_initcall(final, 1, fn)
 
+/*
+ * These nex_* init-calls are provided for drivers and services that
+ * resides in the nexus in case of virtualization.  The init-calls are
+ * performed before exiting to at the end of boot initialization. In case
```

Comment:
s/exiting to/exiting to non-secure world/ ?

Reply:
Yes, thanks.